### PR TITLE
Update CSP for ads

### LINKED ISSUE #7366: Ensure Live and Test CSP is correct for Ads
**Is your feature request related to a problem? Please describe.**
We have implemented some changes to our content security policy for ads. We should ensure before going live that we test each service, and do not see any content security policy errors. If we do, we should ensure that the CSP is corrected.

This needs to be tested for both Canonical and AMP implementations.

Also test this using the querystring to use the live GNL script and flag any issues.

As a follow up to this, we will monitor CSP violations during the rollout to live. As we can release service by service using iSite, we can release the smallest service first, and disable ads while we update our CSP header if we encounter any live CSP violations that weren't picked up beforehand.

For example, we currently see the following on test:

```
Refused to load the image 'https://googleads.g.doubleclick.net/pagead/ide_cookie' because it violates the following Content Security Policy directive: "img-src https://ichef.bbci.co.uk https://ichef.test.bbci.co.uk https://ping.chartbeat.net https://a1.api.bbc.co.uk/hit.xiti https://logws1363.ati-host.net https://news.files.bbci.co.uk https://news.test.files.bbci.co.uk https://r.bbci.co.uk https://pagead2.googlesyndication.com https://securepubads.g.doubleclick.net https://tpc.googlesyndication.com https://www.google.com http://b.files.bbci.co.uk http://ping.chartbeat.net https://syndication.twitter.com https://platform.twitter.com https://pbs.twimg.com https://i.ytimg.com https://ton.twimg.com https://news.bbcimg.co.uk https://static.bbc.co.uk http://static.bbc.co.uk https://collector.effectivemeasure.net https://csi.gstatic.com https://sb.scorecardresearch.com https://secure-us.imrworldwide.com data: 'self'".
```

**Describe the solution you'd like**
Add any missing entries to the live and test CSP.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by tochwill**
Resolves #7366

**Overall change:**
This updates the test content security policy for ads based on the most up to date feeback from GNL, and brings the live CSP in line with it before release.

**Code changes:**

- CSP entries for ads have been separated from the rest of the CSP entries, to be more easily reused across the different environments.
- Unit tests have been updated.

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/server/utilities/cspHeader/index.js [hunk 1]
```diff
@@ -9,14 +9,66 @@ import getOriginContext from '#contexts/RequestContext/getOriginContext';
  * View the developer console for errors.
  */
 
+const advertisingDirectives = {
```
Comment (tochwill) [inline src/server/utilities/cspHeader/index.js:12]: I think as a follow on to this we could start looking at creating a different CSP for different page types? We don't need these for all pages, we don't need includes for all pages etc, there's potentially more cases like this?

Comment (DenisHdz) [reply src/server/utilities/cspHeader/index.js:12]: That sounds like a good idea 👍

**Comment by tochwill** [comment]
Note to self - are we in danger of hitting the header length limit again with these changes?

**Review by mmcculloch — APPROVED** [review]
LGTM 👍  TravisCI is being a bit weird, but might fix itself

**Review by DenisHdz — APPROVED** [review]
LGTM

**Comment by PriyaKR** [comment]
Looks good to me locally on chrome.Can we merge it and test across more browsers on test env?

**Comment by tochwill** [comment]
I'm just double checking max size limits before we merge this in.

**Comment by tochwill** [comment]
The header size should still be well below our limits as defined in nginx, so we can put this on test now.

**Pull request merged**